app/controllers/api/subcollections/cloud_tenants.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module Subcollections
+    module CloudTenants
+      def cloud_tenants_query_resource(object)
+        object.respond_to?(:cloud_tenants) ? object.cloud_tenants : []
```

Comment:
I think the potential is definitely there if we discover we want to mix this into more places.  I know that accommodating for cases like this is a pretty big deal in the UI, so using respond_to felt safer!